apps/govquests-frontend/src/app/page.tsx

```diff
@@ -1,101 +1,52 @@
-          </a>
-          <a
-            className="rounded-full border border-solid border-black/[.08] dark:border-white/[.145] transition-colors flex items-center justify-center hover:bg-[#f2f2f2] dark:hover:bg-[#1a1a1a] hover:border-transparent text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:min-w-44"
-            href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Read our docs
-          </a>
-        </div>
-      </main>
-      <footer className="row-start-3 flex gap-6 flex-wrap items-center justify-center">
-        <a
-          className="flex items-center gap-2 hover:underline hover:underline-offset-4"
-          href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <Image
-            aria-hidden
-            src="https://nextjs.org/icons/file.svg"
-            alt="File icon"
-            width={16}
-            height={16}
-          />
-          Learn
-        </a>
-        <a
-          className="flex items-center gap-2 hover:underline hover:underline-offset-4"
-          href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <Image
-            aria-hidden
-            src="https://nextjs.org/icons/window.svg"
-            alt="Window icon"
-            width={16}
-            height={16}
-          />
-          Examples
-        </a>
-        <a
-          className="flex items-center gap-2 hover:underline hover:underline-offset-4"
-          href="https://nextjs.org?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <Image
-            aria-hidden
-            src="https://nextjs.org/icons/globe.svg"
-            alt="Globe icon"
-            width={16}
-            height={16}
+  const router = useRouter();
+
+  useEffect(() => {
+    const fetchQuests = async () => {
+      try {
+        const data = await getQuests();
+        setQuests(data);
+        setLoading(false);
+        console.log(data);
+      } catch (err) {
+        setLoading(false);
+      }
+    };
+
+    fetchQuests();
+  }, []);
+
+  if (loading) return <div>Loading...</div>;
+  return (
+    <main className="p-6 flex-1">
+      <h2 className="text-2xl">Here's some quests for you!</h2>
+      <span className="text-lg bg-green mb-4">
+        We selected them based on your profile
+      </span>
+      <div className="mt-6 grid grid-cols-2 md:grid-cols-3 lg:grid-cols-4 2xl:grid-cols-5 gap-5">
+        {quests.map((quest) => (
+          <Quest
```

Comment:
I would wrap this into a next.js Link (https://nextjs.org/docs/pages/api-reference/components/link) component instead of using `onClick={() => router.push("1")}`

Reply:
So we can reuse the question component if needed and take advantage of the features that the nextjs link component has